File: pyhton sorts code/selectionSort.py
def selectionSort(arr):
    n = len(arr)
    for i in range(n - 1):
        minIdx = i
        for j in range(i + 1, n):
            if arr[j] < arr[minIdx]:
                minIdx = j
        arr[i], arr[minIdx] = arr[minIdx], arr[i]

def printArray(arr):
    for num in range(len(arr)):
        print(arr[num],end="\t")
    print()

arr = [60, 40, 50, 30, 10, 20]
print("Array before selection sort:", end=" ")
printArray(arr)
selectionSort(arr)
print("Array after selection sort:", end=" ")
printArray(arr)

File: cpp sorts codes/selectionSort.cpp
#include <iostream>
#include<algorithm>
using namespace std;
void selectionSort(int* arr, int n) {

	int mianIdx;
	for (int i = 0; i < n - 1; i++) {
		mianIdx = i;
		for (int j = i + 1; j < n; j++) {
			if (arr[j] < arr[mianIdx]) {
				mianIdx = j;
				
			}
		}
		swap(arr[mianIdx], arr[i]);
	}
}
void print(int* arr, int n) {

	for (int i = 0; i < n; i++) {
		cout << arr[i] << "\t";
	}
	cout << endl;
}
int main()
{
	int arr[] = { 60,40,50,30,10,20 };
	int n = sizeof(arr) / sizeof(arr[0]);
	cout << "Array before selection sort : ";
	print(arr, n);
	selectionSort(arr, n);
	cout << "Array after selection sort : ";
	print(arr, n);

}

